var/spack/repos/builtin/packages/py-merlin-info/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyMerlinInfo(PythonPackage):
+    """A custom version of Phillip J. Eby's setuptools."""
+
+    homepage = "https://pypi.org/project/merlin_info/"
```

Comment:
This homepage doesn't exist, remove it

Reply:
If the url doesn't work then this package is not useful and should be removed.